mail/pfSense-pkg-mailreport/files/etc/inc/mail_reports.inc

```diff
@@ -80,6 +80,9 @@ function mail_report_send($headertext, $cmdtext, $logtext, $attachments) {
 
 	if (empty($config['notifications']['smtp']['ipaddress']))
 		return;
+	
+	if (empty($config['notifications']['smtp']['notifyemailaddress']))
```

Comment:
Please use ["K&R, BSD KNF variant style" formatting](https://www.netgate.com/docs/pfsense/development/developer-style-guide.html#indent-style). I know you were just matching what was already there, but it's better to do it right.

You can fix the other existing code if you like, in a separate commit.